src/main/kotlin/no/nav/familie/ef/personhendelse/client/pdl/PdlClient.kt

```diff
@@ -57,4 +57,4 @@ private fun httpHeadersPdl(): HttpHeaders {
 
 fun String.graphqlCompatible(): String {
     return StringUtils.normalizeSpace(this.replace("\n", ""))
-}
\ No newline at end of file
+}
```

Comment:
Hva er det egentlig som skjer med alle endringene på siste linje, på alle filer?

Reply:
Rart det ikke vises bedre, men det legges til en tom linje på slutten. Søkte opp hvorfor, og grunnen til at det er satt opp er delvis historisk og delvis pga diffing slik jeg tolker det: 
https://stackoverflow.com/questions/6549452/new-line-at-the-end-of-source-code
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file
men det er vel ikke noe som treffer oss.

Reply:
Da skjønner jeg :)